### PR TITLE
Bug 1639136 - Endpoints related to alert summaries should retrieve performance tags

### PR DESCRIPTION
**Pull request by esanuandra**

### CODE REVIEW

File: treeherder/webapp/api/performance_serializers.py [hunk 1]
```diff
@@ -178,9 +179,16 @@ class Meta:
         ]
 
 
+class PerformanceTagSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = PerformanceTag
+        fields = ['id', 'name']
```
Comment (ionutgoldan) [inline treeherder/webapp/api/performance_serializers.py:185]: We should ensure the `name` field is read only.

File: treeherder/webapp/api/performance_serializers.py [hunk 2]
```diff
@@ -178,9 +179,16 @@ class Meta:
...
+
+
 class PerformanceAlertSummarySerializer(serializers.ModelSerializer):
     alerts = PerformanceAlertSerializer(many=True, read_only=True)
     related_alerts = PerformanceAlertSerializer(many=True, read_only=True)
+    performance_tags = PerformanceTagSerializer(many=True, read_only=True)
```
Comment (ionutgoldan) [inline treeherder/webapp/api/performance_serializers.py:191]: Why read only here? We need to allow users to update a summary's perf tags.

File: tests/webapp/api/test_performance_alertsummary_api.py [hunk 3]
```diff
@@ -89,6 +89,7 @@ def test_alert_summaries_get(client, test_perf_alert_summary, test_perf_alert):
             'revision',
```
Comment (ionutgoldan) [inline tests/webapp/api/test_performance_alertsummary_api.py:89]: We need to add some end-to-end backend tests. To ensure that the user can: 
* add multiple tags to a summary
* remove a tag from a summary
* not add an unregistered perf tag to a summary

File: treeherder/webapp/api/performance_serializers.py [hunk 4]
```diff
@@ -178,9 +179,18 @@ class Meta:
...
+
+    class Meta:
+        model = PerformanceTag
+        fields = ['id', 'name']
+
+
 class PerformanceAlertSummarySerializer(serializers.ModelSerializer):
     alerts = PerformanceAlertSerializer(many=True, read_only=True)
     related_alerts = PerformanceAlertSerializer(many=True, read_only=True)
+    performance_tags = PerformanceTagSerializer(many=True, required=False)
```
Comment (ionutgoldan) [inline treeherder/webapp/api/performance_serializers.py:193]: Shouldn't we be using a `SlugRelatedField` here? Or have I missed something?

Comment (esanuandra) [reply treeherder/webapp/api/performance_serializers.py]: Yes, we should, the pr is updated now

**Pull request merged**